src/views/App/index.tsx

```diff
@@ -126,6 +130,20 @@ const AppRoutes = () => {
       {flags.systemProfile && (
         <SecureRoute exact path="/system-profile" component={SystemList} />
       )}
+      {flags.systemProfile && (
+        <SecureRoute
+          path="/system-profile/:systemId"
+          exact
+          component={SystemProfile}
+        />
+      )}
+      {flags.systemProfile && (
+        <SecureRoute
+          path="/system-profile/:systemId/:subinfo/:top?"
```

Comment:
For some reason when I search for and click `medicare.gov` in the System list, I am brought to http://localhost:3000/system-profile/326-510-0/home/top which gives me a "page not found" error. Can't seem to figure out exactly what's going on here.

Reply:
Ah - I think this is because it's pulling from `mockSystemInfo` still - right? Didn't catch that my first pass through.

Reply:
I'm sorry I forgot to mention in the PR instructions that only the first table item (1-800 Medicare...) can be viewed on system profile, due to the mock data needing to match up.  We haven't implemented a single system query for system profile, so it needs to use mock data.  The mocked data id needs to correspond with the clicked table item, and currently it's only set up to do the first.

Reply:
Oh and the double header was resolved after the merge conflict with Adam's PR